components/atom/input/src/Input/Component/index.js

```diff
@@ -85,6 +88,8 @@ Input.propTypes = {
   placeholder: PropTypes.string,
   /* 's' or 'm', default: 'm' */
   size: PropTypes.oneOf(Object.values(SIZES)),
+  /* width of input based in number of characters (native "size" attribute) */
+  sizeChars: PropTypes.number,
```

Comment:
I'd say charsSize in proper english.
(Also, as its the same feature as `cols` in textarea, we could use it as alternative name of prop.)